org.kryogenix.utm_no.yml

```diff
@@ -0,0 +1,121 @@
+app-id: org.kryogenix.utm_no
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+command: utm_no
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --talk-name=org.kde.StatusNotifierWatcher
+
+rename-desktop-file: utm_no.desktop
+rename-appdata-file: utm_no.metainfo.xml
+rename-icon: utm_no
+
+modules:
+  - name: utm_no
+    buildsystem: simple
+    build-commands:
+      - install -Dm755 flatpak/launcher.sh /app/bin/utm-no
+      - install -Dm755 flatpak/launcher.sh /app/bin/utm_no
```

Comment:
the thing is that the commanline here isn't really for public use. From the command line you'd start the app with `flatpak run org.kryogenix.utm_no`

Reply:
oh! fair enough. I thought that since I declared the binary name in appstream, that it got exposed somehow. I'll remove one of them then! Cheers for that pointer.

Reply:
Nope, not at all - the *only* things that get exported to the host system are named after your app-id. There's certainly some downsides to this but it does prevent some of the more obvious bad behaviour.

Reply:
fixed in latest push.